semgrep-core/tests/dockerfile/add-chown.dockerfile

```diff
@@ -0,0 +1,6 @@
+ADD file1 file2 /mydir/
```

Comment:
usually the semgrep tests files in semgrep-core/tests/xxx/ follow certain conventions. See the script that build the cheatsheet. I would recomment to rename lots of those files misc_xxx.dockerfile to not clutter the different test files that exhibit important features of semgrep (e.g., metavar_args.dockerfile, dots_args.dockerfile, etc.).

Reply:
Sure, but each docker command has its own syntax and needs to be tested. They're all converted to functions in the generic AST. I think I'll use the standard test names for the `RUN` commands since they are most commonly used.